index.html

```diff
@@ -2022,73 +2022,62 @@
 they're interacting with. [=Actors=] can be [=people=] or collective entities like companies,
-associations, or governmental bodies. Uses of this document in a particular domain are expected to
-describe how the core concepts of that domain combine into a [=user=]-comprehensible [=actor=], and
-those refined definitions are likely to differ between domains.
+associations, or governmental bodies.
 
 [=User agents=] tend to explain to [=people=] which [=origin=] or [=site=] provided the
-web page they're looking at. The [=actor=] that controls this [=origin=] or [=site=] is
+web page they're looking at. The [=actor=] that makes or delegates decisions
+about the content and [=data processing=] on this [=origin=] or [=site=] is
 known as the web page's <dfn data-dfn-for="page">first party</dfn>. When a [=person=]
-interacts with a UI element on a web page, the <dfn>first party</dfn> of that interaction
-is usually the web page's [=page/first party=]. However, if a different [=actor=] controls
-how data collected with
-the UI element is used, and a reasonable person with a realistic cognitive budget would realize
+interacts with a part of a web page, the <dfn>first party</dfn> of that interaction
+is usually the web page's [=page/first party=].
+However, if a different [=actor=] makes the decisions about how that part of the
+page works, and a reasonable person with a realistic cognitive budget would realize
 that this other [=actor=] has this control, this other
 [=actor=] is the [=first party=] for the interaction instead.
 
-<aside class="issue">
-  The group believes that the definition of [=first party=] provided above needs further refinement.
-  Please see <a href="https://github.com/w3ctag/privacy-principles/issues/152">this issue</a>
-  for more details.
-</aside>
-
-The [=first party=] to an interaction is accountable for the processing of data produced
-by that interaction, even if another actor does the processing.
+If someone captures data about an interaction with a web page,
+the [=first party=] of that interaction is accountable for the way that data is [=processed=],
+even if another [=actor=] does the processing.
 
 A <dfn data-lt="third parties">third party</dfn> is any [=actor=] other than the
 [=person=] visiting the website or the [=first parties=] they expect to be interacting
 with.
 
-The <dfn>Vegas Rule</dfn> is a simple implementation of privacy in which "<i>what happens with the
-[=first party=] stays with the [=first party=]</i>." Put differently, the [=Vegas Rule=] is followed
-when the [=first party=] is the only [=data controller=]. While the [=Vegas Rule=] is a good
-guideline, it's neither necessary nor sufficient for [=appropriate=] [=data processing=]. A [=first
-party=] that maintains exclusive access to a person's data can still [=process=] it
-[=inappropriately=], and there are cases where a third party can learn information about a person
-but still treat it [=appropriately=].
-
```

Comment:
Can we not revisit decisions we've discussed and made? See:
* https://github.com/w3ctag/privacy-principles/pull/77
* https://github.com/w3ctag/privacy-principles/blob/48f13c9b0110f9422041f8e0929711b27e793469/meetings/2021-11-17-minutes.md

Reply:
We re-discussed this today and decided to keep the Vegas Rule out.